app/assets/javascripts/admin/controllers/admin-customize-themes-show.js.es6

```diff
@@ -9,6 +9,12 @@ const THEME_UPLOAD_VAR = 2;
 export default Ember.Controller.extend({
   editRouteName: "adminCustomizeThemes.edit",
 
+  @computed("selectableChildThemes")
+  selectedChildThemeId() {
+    const available = this.get("selectableChildThemes");
+    return available && available[0] ? available[0].get("id") : null;
```

Comment:
this is confusing code, why is it picking first?

Reply:
This is a fix for this bug: https://meta.discourse.org/t/86399

The repro for this bug is to turn on "Theme includes other child themes" on a theme, you'll see the dropdown menu for components has the first item selected. Now if you don't open the dropdown menu and select a component from it, but instead you click Add right away, you'll see this bug.

`selectedChildThemeId` is controlled by the dropdown menu, and it's undefined until you select something from the dropdown menu which is causing this bug. I added this to set an initial value to the property.

I think there is a clearer way to fix this bug, let me try it.